client/app/constants/SearchPageConstants.js

```diff
@@ -0,0 +1,6 @@
+// Search page
+const CURRENT_PAGE_UPDATE = 'SEARCH::CURRENT_PAGE_UPDATE';
+
+export {
+  CURRENT_PAGE_UPDATE,
+};
```

Comment:
We have such a few constants still that splitting those into multiple files is IMO cumbersome. Could they all live in a single file where you can easily see all possible action types?